include/javascript/mozaik/jquery.mozaik.css

```diff
@@ -230,4 +230,22 @@ div.mozaik-preloader {
 .mozaik-thumbs {
     background: #fff;
     height: 400px;
+}
+
+/* margin set */
+.mozaik-width-set {
```

Comment:
It would be better, if the properties were in alpha-numeric order. It makes it easier for next person who looks at it